sources/packages/api/src-sql/EducationPrograms/Rollback-alter-eduction-programs.sql

```diff
@@ -0,0 +1,77 @@
+--Set NOT NULL constraint on program_description
+ALTER TABLE
```

Comment:
I believe that this would fail if there are null values in the column. Maybe would be good to d an update to some default value when is null.

Reply:
Good catch. Will update it.

Reply:
Default value added.